src/applications/disability-benefits/2346/components/Batteries.jsx

```diff
@@ -198,9 +205,10 @@ class Batteries extends Component {
                   status="warning"
                 />
               ) : (
-                <div>
+                <div className="vads-u-width-293">
```

Comment:
Does this actually apply a 293 pix width ? We should avoid using hard pixel counts like this.  If anything we can use pixel counts for max widths

Reply:
Yes this class a 293px width. I can refactor it to make it a max-width of 293px.

Reply:
Hmm so the battery and accessory page input divs need a width attached so the input can copy its size and not make the click zone the whole width of the row. That's also why I added the `.address-page, .battery-page, .accessory-page {
  [type=checkbox], [type=radio] {
    width: inherit;
  }
}` code.

Reply:
Check it on mobile to ensure it looks ok there too

Reply:
I refactored it out some more and got rid of the `width: inherit` code so it's just using a max width of 293 and 226.